questionnaire/forms.py

```
@@ -1,15 +1,16 @@
 from django import forms
 from crispy_forms.helper import FormHelper
-from crispy_forms.layout import Field, Layout, HTML,Div
+from crispy_forms.layout import Field, Layout, HTML, Div
 from questionnaire.models import QuestionnaireData
 from django.utils.translation import gettext_lazy as _
```

Ruff (0.12.2)

`django.utils.translation.gettext_lazy` imported but unused

Remove unused import: `django.utils.translation.gettext_lazy`

(F401)